app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -37,6 +37,29 @@
   <% end %>
 <% end %>
 
+<% unless FeatureFlags::FeatureFlag.active?(:application_work_history) && !region.application_form_skip_work_history %> 
+  <p class="govuk-body"><%= proof_of_recognition_description_for(region:) %></p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>school name</li>
+    <li>school city</li>
+    <li>school country</li>
+    <li>your job role</li>
+    <li>start date</li>
+    <li>end date</li>
+  </ul>
+<% end %>
+
+<% if region.sanction_check_written? %>
+  <p class="govuk-body"><%= t("eligibility_interface.three_months_before_applying") %></p>
```

Comment:
Does this sentence belong above the work history as well? It looks like it's more connected to the written statement than the work history.